widgetastic_manageiq.py

```diff
@@ -878,3 +875,57 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class AlertEmail(Widget):
+
+    ROOT = "//div[@id='edit_email_div']"
+    ADD_BUTTON = ".//div[@title='Add']"
+    RECIPIENTS = "./div[@id='edit_to_email_div']//a"
+    EMAIL_LINK = ".//a[text()='{}']"
+
+    def __init__(self, parent, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.RECIPIENTS_INPUT = TextInput(self, "email")
```

Comment:
Me no gusta, this may lead to problems. Widgets are instantiated lazily on request.